scripts/zkai_native_attention_mlp_rmsnorm_label_sensitivity_gate.py

```
@@ -0,0 +1,852 @@
+        case_obj = _dict(case, f"mutation case {index}")
+        name = case_obj.get("name")
+        if not isinstance(name, str):
+            raise RmsnormLabelSensitivityError("mutation case name drift")
+        names.append(name)
+        if case_obj.get("rejected") is not True:
+            raise RmsnormLabelSensitivityError(f"mutation not rejected: {name}")
+    if tuple(names) != EXPECTED_MUTATION_NAMES:
+        raise RmsnormLabelSensitivityError("mutation inventory drift")
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (46 > 12)

(PLR0912)